Packs/SecneurXAnalysis/Integrations/SecneurXAnalysis/SecneurXAnalysis.py

```diff
@@ -0,0 +1,611 @@
+                outputs_prefix="SnXAnalysis.SubmitFile",
+                outputs=finalJson
+            )
+        else:
+            readableOutput = tableToMarkdown("File Submission Failed: {fileName}", response)
+            return CommandResults(
+                readable_output=readableOutput,
+                outputs_prefix="",
+                outputs=response
+            )
+    else:
+        msg = error_response(err_msg)
+        raise DemistoException(msg)
+
+
+def post_submit_url(client: Client, args: Dict[str, str]) -> CommandResults:
+    urlSuffix = "/analyze_url"
+    urlValue = args.get("URL") or None
+    if urlValue is None or len(urlValue) == 0:
+        raise DemistoException("Input url value is empty")
+    params = create_request_json(args)
+    urlParams = {SNXReportParser.JSON_URL: urlValue}
+    response, err_msg = client.submit_url(urlSuffix, params, urlParams)
+    if response:
+        if SNXResponse.SNX_SUCCESS_KEY in response.keys() and SNXResponse.SNX_RESULT_KEY in response.keys():
+            finalJson = response[SNXResponse.SNX_RESULT_KEY]
+            readableOutput = tableToMarkdown("URL Submitted Successfuly", finalJson)
+            return CommandResults(
+                readable_output=readableOutput,
+                outputs_prefix="SnXAnalysis.SubmitURL",
+                outputs=finalJson
+            )
+        else:
+            readableOutput = tableToMarkdown("URL Submission Failed", response)
+            return CommandResults(
+                readable_output=readableOutput,
+                outputs_prefix="",
+                outputs=response
+            )
+    else:
+        msg = error_response(err_msg)
+        raise DemistoException(msg)
+
+
+def get_verdict_cmd(client: Client, args: Dict[str, str]) -> CommandResults:
+    taskUuid = args.get(SNXResponse.SNX_TASK_UUID_KEY) or None
+    if taskUuid is None:
+        raise DemistoException("Task UUID Parameter value is not found")
+    else:
+        urlSuffix = "/get_verdict"
+        params = {SNXResponse.SNX_TASK_UUID_KEY: taskUuid}
+        response, err_msg = client.get_response(urlSuffix, params)
+        if response:
+            entryContent = {}
+            finalJson = {}
+            if SNXResponse.SNX_SUCCESS_KEY in response.keys() and SNXResponse.SNX_RESULT_KEY in response.keys():
+                finalJson = response[SNXResponse.SNX_RESULT_KEY]
+                entryContent = {
+                    "SnXAnalysis.Verdict(val.task_uuid == obj.task_uuid)": finalJson
+                }
+            else:
+                finalJson = response
+                entryContent = response
+            readableOutput = tableToMarkdown(f"SecneurX Analysis - Verdict Result: {taskUuid}", t=finalJson)
+            return CommandResults(
+                readable_output=readableOutput,
+                outputs=entryContent,
+                outputs_prefix="",
+                raw_response=finalJson
+            )
```

Comment:
In case the condition in line `369` is not met,
The `outputs` will not be under a special key.
I suggest returning different commandResults in case the condition is met and in case the condition is not met.

like this:

```suggestion
            if SNXResponse.SNX_SUCCESS_KEY in response.keys() and SNXResponse.SNX_RESULT_KEY in response.keys():
                finalJson = response[SNXResponse.SNX_RESULT_KEY]
                return CommandResults(
                    readable_output=readableOutput,
                    outputs=finalJson,
                    outputs_key_field="task_uuid",
                    outputs_prefix="SnXAnalysis.Verdict",
                    raw_response=finalJson
                    )
            else:
                finalJson = response
                readableOutput = tableToMarkdown(f"SecneurX Analysis - Verdict Result: {taskUuid}", t=finalJson)
                return CommandResults(
                    readable_output=readableOutput,
                    outputs=finalJson,
                    outputs_prefix="SnXAnalysis.Verdict",
                    raw_response=finalJson
                    )
```